src/plugins/notebook/components/NotebookEmbed.vue

```diff
@@ -40,14 +40,20 @@ export default {
     components: {
         PopupMenu
     },
-    inject: ['openmct'],
+    inject: ['openmct', 'snapshotContainer'],
     props: {
         embed: {
             type: Object,
             default() {
                 return {};
             }
         },
+        isSnapshotContainer: {
```

Comment:
Since you're providing the `snapshotContainer` to this view, won't `isSnapshotContainer` always be true? Or is that prop named incorrectly?